### PR TITLE
Add: Toggle to enable EPSS fields

### PR DESCRIPTION
**Pull request by timopollmeier**
## What
The EPSS fields for CVEs and VTs can now be toggled with an option in the config.js file.

## Why
This is done because no EPSS feed is provided yet, so the new fields should only be enabled for testing.

## References
GEA-571

## Checklist

- [x] Tests

### CODE REVIEW

File: src/web/pages/nvts/table.jsx [hunk 1]
```diff
@@ -124,29 +127,38 @@ const Header = ({
...
-          currentSortDir={currentSortDir}
-          currentSortBy={currentSortBy}
-          sortBy={sort ? 'epss_percentile' : false}
-          onSortChange={onSortChange}
-          title={_('Percentile')}
-        />
-      </TableRow>
+      {
+        gmp.settings.enableEPSS &&
+        <>
```
Comment (daniele-mng) [inline src/web/pages/nvts/table.jsx:140]: Nitpick: I think we can remove the  wrapper fragment.

File: src/web/pages/nvts/row.jsx [hunk 2]
```diff
@@ -102,14 +104,19 @@ const Row = ({
...
-      </TableData>
-      <TableData>
-        {isNumber(entity?.epss?.max_severity?.percentile)
-          ? entity.epss?.max_severity?.percentile.toFixed(5) : _("N/A")}
-      </TableData>
+      {
+        gmp.settings.enableEPSS &&
+        <>
+          <TableData>
+            {isNumber(entity?.epss?.max_severity?.score)
```
Comment (daniele-mng) [inline src/web/pages/nvts/row.jsx:111]: Question: Not sure what convention we have in the codebase for in lining, but maybe if we move it to a `const` could be easier to understand with the name?

File: src/web/pages/cves/details.jsx [hunk 3]
```diff
@@ -25,6 +25,7 @@ import TableData from 'web/components/table/data';
 import TableRow from 'web/components/table/row';
 
 import PropTypes from 'web/utils/proptypes';
+import useGmp from "web/utils/useGmp.jsx";
```
Comment (bjoernricks) [inline src/web/pages/cves/details.jsx:28]: Imports should not use an extension. @daniele-mng could we have a eslint rule for that too?

```suggestion
import useGmp from "web/utils/useGmp";
```

File: src/web/pages/cves/row.jsx [hunk 4]
```diff
@@ -26,44 +26,55 @@ import {RowDetailsToggle} from 'web/entities/row';
 
 import PropTypes from 'web/utils/proptypes';
 import {isNumber} from "gmp/utils/identity";
+import useGmp from "web/utils/useGmp.jsx";
```
Comment (bjoernricks) [inline src/web/pages/cves/row.jsx:29]:
```suggestion
import useGmp from "web/utils/useGmp";
```

File: src/web/pages/cves/table.jsx [hunk 5]
```diff
@@ -21,6 +21,7 @@ import TableRow from 'web/components/table/row';
 import CveDetails from './details';
 import CveRow from './row';
 import {isDefined} from "gmp/utils/identity.js";
+import useGmp from "web/utils/useGmp.jsx";
```
Comment (bjoernricks) [inline src/web/pages/cves/table.jsx:24]:
```suggestion
import useGmp from "web/utils/useGmp";
```

File: src/web/pages/nvts/details.jsx [hunk 6]
```diff
@@ -12,6 +12,7 @@ import {isDefined, isNumber} from 'gmp/utils/identity';
 import {TAG_NA} from 'gmp/models/nvt';
 
 import PropTypes from 'web/utils/proptypes';
+import useGmp from "web/utils/useGmp.jsx";
```
Comment (bjoernricks) [inline src/web/pages/nvts/details.jsx:15]:
```suggestion
import useGmp from "web/utils/useGmp";
```

File: src/web/pages/nvts/row.jsx [hunk 7]
```diff
@@ -30,6 +30,7 @@ import EntitiesActions from 'web/entities/actions';
 import {RowDetailsToggle} from 'web/entities/row';
 
 import PropTypes from 'web/utils/proptypes';
+import useGmp from "web/utils/useGmp.jsx";
```
Comment (bjoernricks) [inline src/web/pages/nvts/row.jsx:33]:
```suggestion
import useGmp from "web/utils/useGmp";
```

File: src/web/pages/nvts/table.jsx [hunk 8]
```diff
@@ -27,6 +27,8 @@ import TableRow from 'web/components/table/row';
 import NvtDetails from './details';
 import NvtRow from './row';
 
+import useGmp from "web/utils/useGmp.jsx";
```
Comment (bjoernricks) [inline src/web/pages/nvts/table.jsx:30]:
```suggestion
import useGmp from "web/utils/useGmp";
```

**Review by bjoernricks — COMMENTED** [review]
The new setting should be documented in the README too.

**Pull request merged**